docs/DeveloperGuide.md

```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Customise Order Feature
+
+#### Implementation
+
+This feature has 2 crucial methods for this feature are in the `CustomiseCommand` and `PersonCard` classes.
```

Comment:
Better phrasing? `This feature has 2 crucial methods for this feature are in...` is pretty confusing to read

Reply:
Ok, I have changed it.